### ISSUE TITLE
Wildcards as a source parameter

### ISSUE THREAD

**Issue by drone1** [HeroProtagonist/recursive-rename#4]
I have a directory filled with extensionless files which are all meant to be aiffs.

If I do:

```rename * aif```

rename prints nothing, and nothing is affected.

I'm in zsh (but also tried in bash for the hell of it, since zsh can be weird about wildcards). I also tried ```rename "*" aif``` (quotes around the wildcard), but it appears to be a no-op.

Would love to be able to use this as it would save me a bunch of time, so please let me know if I'm doing something wrong! Thanks a lot.

**Comment by HeroProtagonist** [comment]
Hi @drone1,

I assume that files inputted have extensions. I like the idea of adding `*` as a wild card to change extension of all files. A PR with this change would be welcome. Otherwise, I could do this later this week.

Cheers!

**Comment by drone1** [comment]
No, my files did not have extensions and that was most likely the problem.

I have a bunch of audio files which somehow lost their extensions when
transferred across file systems or something. So I was trying to
recursively add extensions to all files.

I ended up writing a bash script to do it. Bleh.
